CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Changelog
 
+## 1.8.0 (2021-07-07)
+
+- Use webpack for logging (#55)
+- Upgrade build reporter to latest version (#57)
+- Upgrade source map uploader to latest version (#60)
```

Comment:
I don't think this should be in the changelog — it was only a lockfile update so nothing has changed for users